handlers/svg-open-street-map/osm-svg.py

```diff
@@ -75,186 +123,210 @@ def handle():
-                node_coordinates = [[node["lon"], node["lat"]]
-                                    for node in streets[street]["nodes"]]
-                for index in range(len(node_coordinates) - 1):
-                    p.M(scaled_longitude *
-                        (node_coordinates[index][0] -
-                         lon_min), scaled_latitude *
-                        (node_coordinates[index][1] -
-                         lat_min))
-                    p.L(scaled_longitude *
-                        (node_coordinates[index +
-                                          1][0] -
-                         lon_min), scaled_latitude *
-                        (node_coordinates[index +
-                                          1][1] -
-                         lat_min))
-                all_svg.append(p)
-                svg.append(p)
-                if "street_name" in streets[street]:
-                    svg.append(
-                        draw.Text(
-                            streets[street]["street_name"],
-                            14,
-                            path=p,
-                            text_anchor='start',
-                            line_height=1))
-                    svg_layers.append(
-                        {"label": streets[street]["street_name"],
-                            "svg": svg.asDataUri()})
-                    svg = draw.Drawing(dimensions[0], dimensions[1])
-                else:
-                    svg_layers.append(
-                        {"label": str(streets[street]["street_id"]),
-                            "svg": svg.asDataUri()})
-                    svg = draw.Drawing(dimensions[0], dimensions[1])
-            # Draw all points of interest (POIs)
-            if ("points_of_interest" in data or
-                    len(data["points_of_interest"]) != 0):
-                for points_of_interest in data["points_of_interest"]:
-                    if points_of_interest["cat"] != "intersection":
-                        latitude = (
-                            (points_of_interest["lat"] - lat_min)
-                            * scaled_latitude)
-                        longitude = (
-                            (points_of_interest["lon"] - lon_min)
-                            * scaled_longitude)
-                        all_svg.append(
-                            draw.Circle(
-                                longitude,
-                                latitude,
-                                3.5,
-                                fill='red',
-                                stroke_width=1.5,
-                                stroke='red'))
-                        all_svg.append(
-                            draw.Text(
-                                points_of_interest["cat"],
-                                16,
-                                longitude,
-                                latitude,
-                                fill='black'))
+            svg_layers = []
```

Comment:
Some blocks of code like this one does not seem to align with the changes you've described in your PR. Let us know what else has been done in this PR so we can properly review it please! You only mention the change in debug messages (which are quite nice) but it seems like you've done more than just change the debug messages!

Reply:
Thanks, but nothing seems to be outside of what was described. The changes that look like some block of codes are not new additions, but appear so with the change of order/organization of the existing blocks brought by the introduction of the debug levels. For an example, in line 132 where the check for map or non-map data is taking place, the existing logic there would simply return an invalid JSON response (i.e. for non-map data), which is one major problem we are addressing with this PR, and it is addressed by the lines 134 -148. This however introduces the supposedly block change observed from line 149. A similar case for the debug level set at line 150 (that confirms if map data is found).